projects/packages/account-protection/.phan/baseline.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Given this is a new package, it might be worth just doing away with this baseline.